packages/sku/src/program/commands/test/test.action.ts

```diff
@@ -21,17 +15,8 @@ export const testAction = async (
   await configureProject(skuContext);
   await runVocabCompile(skuContext);
 
-  // https://jestjs.io/docs/configuration#preset-string
-  const jestPreset = 'sku';
-  log(`Using '${jestPreset}' Jest preset`);
-
-  const jestArgv = [...args];
-
-  jestArgv.push('--preset', jestPreset);
-
-  if (isCI) {
-    jestArgv.push('--ci');
+  if (skuContext.testRunner === 'vitest') {
+    vitestHandler({ skuContext, args });
```

Comment:
Do you need to return here so vitest and jest aren't both run?

```suggestion
    vitestHandler({ skuContext, args });
    return;
```